docs/3.api/4.commands/upgrade.md

```diff
@@ -31,6 +31,7 @@ Option | Default | Description
 --- | --- | ---
 `--cwd=<directory>` |  | Specify the working directory, this takes precedence over ROOTDIR (default: `.`)
 `--logLevel=<silent\|info\|verbose>` |  | Specify build-time log level
+`--dedupe` |  | Will deduplicate dependencies but not recreate the lockfile
```

Comment:
Not sure if this is relevant, but it depends on the package manager. It will dedupe if possible, and fall back to lockfile recreation if not supported (e.g. for bun). https://github.com/unjs/nypm/blob/dfb09efd82735733e2a766a58a98ee4f2a062705/src/api.ts#L247